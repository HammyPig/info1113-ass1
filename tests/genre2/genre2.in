ADD COLLECTION books.csv
GENRE fakeTest
genRE fake multi word gENRE EXTRAVAGANZA!!!