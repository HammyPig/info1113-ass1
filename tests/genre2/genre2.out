user: 17 books successfully added.

user: No books with genre fakeTest.

user: No books with genre fake multi word gENRE EXTRAVAGANZA!!!.

user: Ending Library process.
